README.md

```diff
@@ -224,6 +224,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Aman Shrivastava](https://github.com/aman34503)
   - [Aman Upadhyay](https://github.com/AmanxUpadhyay)
   - [Aman Verma](https://github.com/amanpro1)
+  - [Amar Naik] (https://github.com/amar-naik)
```

Comment:
There's an extra space here that should be removed.
```suggestion
  - [Amar Naik](https://github.com/amar-naik)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable